app/helpers/link_to_helper.rb

```diff
@@ -0,0 +1,16 @@
+module LinkToHelper
```

Comment:
This has nothing to do with LinkTo ? It is css class helper.

And as I mentioned, this might already exist. And it does:
https://github.com/3scale/porta/blob/305483501653cbff5dac0d71c21a29adef33a02d/app/helpers/application_helper.rb#L177